src/OSPSuite.Infrastructure.Import/Core/DataFormat/AbstractColumnsDataFormat.cs

```diff
@@ -47,9 +47,45 @@ private double setParameters(IUnformattedData data, IReadOnlyList<ColumnInfo> co
          ExtractGeneralParameters(keys, data, metaDataCategories, ref totalRank);
          ExtractNonQualifiedHeadings(keys, missingKeys, columnInfos, data, ref totalRank);
          setSecondaryColumnUnit(columnInfos);
+         setDimensionsForMappings(columnInfos);
          return totalRank;
       }
 
+      private void setDimensionsForMappings(IReadOnlyList<ColumnInfo> columnInfos)
+      {
+         foreach (var parameter in _parameters.OfType<MappingDataFormatParameter>())
+         {
+            var mappingColumn = parameter.MappedColumn;
+
+            if (mappingColumn?.Unit == null || mappingColumn.Dimension != null)
```

Comment:
why mappingColumn? in the first part of the if but not in the second? this is not logical. Either you fear that mappingColumn is null or you don't fear. But it should always be the same behavior

Reply:
If mappingColumn (I have just renamed it to mappedColumn, I think it makes more sense) is null, then the first part will be true and then the second part will not be evaluated. But I agree it is not fully logic to have it only on the first part. I will add it to the second part even though it is totally useless.